src/components/ResourceDistribution.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+  <div>
+    <h1 class="text-center">
+      Current leadership distribution across brokers per topic
```

Comment:
nit: change to `Current leadership distribution across brokers per topic in {{ group }} group and {{ cluster }} cluster`. I have seen this helps when we directly navigate to another group/cluster when in the same charts tab.

Reply:
After trying this, I find it a bit too long to read and I think we lose visibility here. Would `{{ resource }} distribution across brokers per topic in cluster {{ cluster }} ({{ group }})` be OK for you? IMO this is shorter and clearer

Reply:
ok. sure.